source/style-guide/style/text-formatting.txt

```diff
@@ -4,603 +4,1636 @@
 
-       In RST, apply the ``:menuselection:`` directive.
-     - Select **Start > Control Panel**, and then click the **Mail**
-       icon.
+        * - .. code-block:: rst
 
-       You can find more documentation about RackConnect in the
-       **Community > Discussions > RackConnect** section of the
-       MyMongoDB Portal.
+               Select :guilabel:`Microsoft Exchange` and then click
+               :guilabel:`Next`.
 
-   * - Syntax statements
-     - Monospace
-     - The main command used to change a file’s owner or group is
-       ``chown``. The most common syntax used with ``chown`` is as
-       follows:
+          - Select :guilabel:`Microsoft Exchange` and then click
+            :guilabel:`Next`.
+
+   Role names
+     Format using the ``:authrole:`` or ``atlasrole`` as appropriate
+     for the application. Otherwise, format as regular text.
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+
+               The :authrole:`Global Owner` role has the permissions to
+               create, read, update, and delete resources within
+               multiple designated products where access is granted.
+
+          - The :authrole:`Global Owner` role has the permissions to
+            create, read, update, and delete resources within multiple
+            designated products where access is granted.
+
+   Sequences
+     Format using menu, then field using the ``:menuselection:`` role.
+     and separate levels with :guilabel:`>`.
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+
+               Select :menuselection:`Admin > Backup > Jobs` to review
+               previously run backups.
+
+               Select :menuselection:`Admin > Backup > Groom Priority`
+               to start moving blocks to a different blockstore.
+
+          - Select :menuselection:`Admin > Backup > Jobs` to review
+            previously run backups.
+
+            Select :menuselection:`Admin > Backup > Groom Priority` to
+            start moving blocks to a different blockstore.
+
+
+   Shell, Terminal, or Command Prompt
+     Format using the ``.. code-block:: console`` directive.
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+               :copyable: false
+
+               .. code-block:: console
+
+                  $ grep "ftp" /etc/xinetd.d/*
+                  /etc/xinetd.d/vsftpd:service ftp
+                  /etc/xinetd.d/vsftpd:server = /usr/sbin/vsftpd
+
+          - .. code-block:: console
+
+               $ grep "ftp" /etc/xinetd.d/*
+               /etc/xinetd.d/vsftpd:service ftp
+               /etc/xinetd.d/vsftpd:server = /usr/sbin/vsftpd
+
+   Syntax statements
```

Comment:
From your example, "Linux/Unix commands" might be more clear. Not sure what syntax is being specified here.